code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -180,6 +180,29 @@
 	anchored = TRUE
 	var/id = null
 
+/obj/structure/closet/secure_closet/brig/genpop
+	name = "genpop storage locker"
+	desc = "Used for storing the belongings of genpop's tourists visiting the locals."
+	/// reference to the ID linked to the locker, done by swiping a prisoner ID on it
+	var/obj/item/card/id/advanced/prisoner/assigned_id
```

Comment:
You may want to make this a weakref to avoid hard deletes, or handle cleaning up references when the locker is destroyed